bundles/com.espressif.idf.core/src/com/espressif/idf/core/build/ESP32CMakeToolChainProvider.java

```diff
@@ -19,7 +19,7 @@ public class ESP32CMakeToolChainProvider extends AbstractESPCMakeToolChainProvid
 	@Override
 	protected IToolChain getToolchain() throws CoreException
 	{
-		return tcManager.getToolChain(ESPToolChainProvider.ID, ESP32ToolChain.ID);
+		return tcManager.getToolChain(ESPToolChainProvider.ID, ESP32ClangToolChain.ID);
 	}
```

Comment:
Maybe this will return esp32clang toolchain for esp32 target? I think which is wrong!

Reply:
fixed